pkg/cli/cmd/resourceprovider/create/create.go

```diff
@@ -46,11 +46,12 @@ Resource providers are the entities that implement resource types such as 'Appli
 
 Creating a resource provider defines new resource types that can be used in applications.
 
-Input can be passed in using a file or inline JSON as the second argument. Prefix the input with '@' to indicate a file path.
+Input can be passed in using a JSON/YAML file as the second argument.
 `,
 		Example: `
 # Create a resource provider
 rad resource-provider create --from-file /path/to/input.yaml
+rad resource-provider create --from-file /path/to/input.json
```

Comment:
Can you separate this to a separate example, and give it it's own heading? As it is, it looks like creating a resource provider requires two commands.